From 7cf74bb6ef2b5f55f9b180d601244f5be8830957 Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Tue, 8 Sep 2015 15:25:42 +0800
Subject: [PATCH 0686/1691] MLK-11520 ARM: imx: enable axi clk in clk_init_on
 on i.mx6qdl

commit 7cf74bb6ef2b5f55f9b180d601244f5be8830957 from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

AXI clk should be always enabled, as many peripheral
devices need this clk to be on, especially for busfreq,
AXI podf change needs AXI clk and its parent on.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 arch/arm/mach-imx/clk-imx6q.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-imx/clk-imx6q.c b/arch/arm/mach-imx/clk-imx6q.c
index 7fe8528..51d4d1f 100644
--- a/arch/arm/mach-imx/clk-imx6q.c
+++ b/arch/arm/mach-imx/clk-imx6q.c
@@ -106,6 +106,7 @@ static unsigned int const clks_init_on[] __initconst = {
 	IMX6QDL_CLK_ROM,
 	IMX6QDL_CLK_ARM,
 	IMX6QDL_CLK_OCRAM,
+	IMX6QDL_CLK_AXI,
 };
 
 static struct clk_div_table clk_enet_ref_table[] = {
-- 
1.9.1

